GettingStarted.md

```diff
@@ -161,24 +292,27 @@ In order to use the Xamarin.Android API in a source file, you will need to add t
   <uses-permission android:name="android.permission.INTERNET" />
   <uses-permission android:name="android.permission.WAKE_LOCK" />
   ```
+3.  Configure required parameters using localytics.xml or through setting options
+	a. Add a localytics.xml in Resources/values. For a sample refer [here](https://github.com/localytics/localytics-xamarin/blob/feature/sdk5.1-android/LocalyticsXamarin/Android/Resources/values/localytics.xml) 
```

Comment:
I would add a note here that says something like:

Any reference in the Localytics documentation to manifest merging will not work in Xamarin as Xamarin does not use gradle as it's underlying build system.  As a result, you should follow any directions that describe installing the SDK manually.